packages.dhall

```diff
@@ -70,4 +70,5 @@ let additions =
         }
       }
 
-in  upstream // additions
+in  (upstream // additions)
+      with node-execa.version = "v1.2.0"
```

Comment:
While testing this locally, I found a bug in `node-execa`'s FFI because I forgot to [add the `return` keyword](https://github.com/JordanMartinez/purescript-node-execa/commit/bf5afc51a8111f8f922fa6a09b3f105802c865d9). This is a situation where being able to parse the JS FFI and warn me that my function returns nothing would have been helpful.